### PR TITLE
feat(UnitsTab): add left padding to 'None highlighted' radio button

### PR DESCRIPTION
**Pull request by assadk88**
## Description

- Add 12px left padding to Box component in UnitsTab (mobile and desktop)
- Maintain existing styling for other radio button variants

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2761--oak-web-application.netlify.thenational.academy
2. Click on \_\_\_\_\_\_
3. You should see \_\_\_\_\_\_

## Screenshots

How it used to look:
image

How it should now look:
image

## Checklist

- [X] Added or updated tests where appropriate
- [X] Manually tested across browsers / devices
- [X] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [ ] Does this PR update a package with a breaking change: No

### CODE REVIEW

File: src/components/CurriculumComponents/UnitsTab/UnitsTab.tsx [hunk 1]
```diff
@@ -249,6 +249,7 @@ const UnitsTab: FC<UnitsTabProps> = ({ trackingData, formattedData }) => {
                     aria-label={"None highlighted"}
                     value={""}
                     data-testid={"no-threads-radio"}
+                    isNoneHighlighted={true}
```
Comment (orangemug) [inline src/components/CurriculumComponents/UnitsTab/UnitsTab.tsx:252]: By adding `isNoneHighlighted` we've made the `<RadioButton/>`'s (SimpleRadioButton) API specific to this component.

Instead we can remove this and add padding to the containing `<Box/>` which I believe is `$pl`

File: src/components/CurriculumComponents/OakComponentsKitchen/SimpleRadio/index.tsx [hunk 2]
```diff
@@ -112,6 +116,7 @@ export function RadioButton(
         alignItems: "center",
         marginBottom: "12px",
         opacity: props.disabled ? 0.4 : 1,
+        ...(props.isNoneHighlighted ? { paddingLeft: "12.8px" } : {}),
```
Comment (orangemug) [inline src/components/CurriculumComponents/OakComponentsKitchen/SimpleRadio/index.tsx:119]: Although this is being removed, we don't generally want to use `px` oak uses `rem`s for old shared components and sizing tokens for new components. The `marginBottom: "12px"` a few lines above is an error.

**Review by orangemug — APPROVED** [review]

**Review by HelenHarlow — APPROVED** [review]
Looks good to me ✅

**Review by rightsaidjames — APPROVED** [review]
Looks good, LGTM

**Comment by rightsaidjames** [comment]
@assadk88 Have changes from a different ticket been added in by mistake?

**Comment by orangemug** [comment]
Yeah @assadk88 looks like
[test: add unit tests for highlightedYearGroupUnitCount function](https://github.com/oaknational/Oak-Web-Application/pull/2761/commits/0451b4a551f448321d2bcd8691a780e18bcef08f) commit has made it's way into this branch

**Comment by assadk88** [comment]
@rightsaidjames @orangemug yes accidentally – I'm looking to revert, sorry!

**Comment by assadk88** [comment]
@rightsaidjames @orangemug those accidentally committed changes have been reverted.

**Review by rightsaidjames — APPROVED** [review]
Thanks, re-approved!

**Pull request merged**